.github/workflows/ci.yml

```
@@ -0,0 +1,121 @@
+name: CI
+
+# Controls when the action will run. Triggers the workflow on push or pull request
+## events but only for the master branch
+on:
+  push:
+    branches:
+      - 'develop'
+  pull_request:
+    branches:
+      - '**'
```

Can we exclude `pyup/**` and `master` here?

```suggestion
      - '**'
      - '!master'
      - '!pyup/**'
```